tests/spread/general/plugin-python-3.6/rockcraft.orig.yaml

```diff
@@ -9,9 +9,9 @@ platforms:
     amd64:
 
 package-repositories:
-  - type: apt
-    ppa: deadsnakes/ppa
-    priority: always
+    - type: apt
```

Comment:
weird, why 4 spaces here but 2 everywhere else?

Reply:
It allows 2 or 4 spaces, but not allow mixed them in one file. I guess I can change it back to 2 spaces

Reply:
I also found there are other 4 spaces yaml files, do you want unify their format?

Reply:
I think if we're going to enable yamllint we should pick a single format, otherwise it's just churn for no good reason imo
I think 2-spaces is the more common standard (used in e.g. github workflows)

Reply:
Fixed to use 2 spaces only